openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -728,7 +728,9 @@ A Client makes a Credential Request to the Credential Endpoint by sending the fo
 
 The `proof_type` parameter is an extension point that enables the use of different types of proofs for different cryptographic schemes.
 
-The `proof` element MUST incorporate the Credential Issuer Identifier (audience), and a `c_nonce` value generated by the Authorization Server or the Credential Issuer to allow the Credential Issuer to detect replay. The way that data is incorporated depends on the key proof type. In a JWT, for example, the `c_nonce` value is conveyed in the `nonce` claim, whereas the audience is conveyed in the `aud` claim. In a Linked Data proof, for example, the `c_nonce` is included as the `challenge` element in the key proof object and the Credential Issuer (the intended audience) is included as the `domain` element.
+The `proof` element MUST incorporate the Credential Issuer Identifier (audience), and a `c_nonce` value generated by the Authorization Server or the Credential Issuer to allow the Credential Issuer to detect replay.  The Wallet MUST use this nonce for subsequent requests until a new nonce is issued by the Credential Issuer. Each issued `c_nonce` value MUST be unique, a Credential Issuer cannot issue duplicate `c_nonces` values.
+
```

Comment:
if we use a nonce more than once we should not use the term nonce. Is mine a wrong assumption or ... ?

Reply:
Agree that the `nonce` name is not the best one, however the multiple usage by the wallet is already allowed by the sentence "The Wallet MUST use this nonce for subsequent requests until a new nonce is issued by the Credential Issuer".
If this multiple usage by the wallet is already allowed, I don't see any advantage on requiring the AS/CI to create unique _nonces_.

Reply:
In my interpretation the text doesn't mandate the use of the same nonce value but the same nonce parameter, since there's the mention to the "new nonce" indicating that in each response a brand new nonce is issued.

I propose to apply normative language to clarify this.

Reply:
i think the current intent is that lifetime of c_nonce is controlled by the issuer, and it is not necessarily unique...